starter-code/app.js

```diff
@@ -5,29 +5,52 @@ const cookieParser   = require("cookie-parser");
 const bodyParser     = require("body-parser");
 const mongoose       = require("mongoose");
 const app            = express();
+const session        = require("express-session");
+const MongoStore     = require("connect-mongo")(session);
+const expressLayouts = require('express-ejs-layouts');
+
+
+const routes = require('./routes/index');
```

Comment:
be consistent, you split your app in 3 routers, but called "controllers" to two of them